ckanext/showcase/commands/migrate.py

```diff
@@ -47,8 +51,8 @@ def command(self):
 
         if cmd == 'migrate':
             self.migrate()
-        elif cmd == 'make_related':
-            self.make_related()
+        elif cmd == 'markdown_to_html':
+            self.markdown_to_html()
```

Comment:
'make_related' is a non-valid command AFAIK since it doesn't exist in this class and it fails to run. So I removed it.